src/api.js

```diff
@@ -5,10 +5,12 @@ baseURL: "https://be-project-nc.onrender.com/api"
 })
 
 export const getArticles = (filterTopic) => {
+    const queryParams = filterTopic ? { params: { topic: filterTopic } } : {};
```

Comment:
Don't need the ternary here. Axios will handle it for you when filterTopic is undefined